src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -86,4 +87,32 @@ private static void setCheckName(InputConfiguration.Builder inputConfigBuilder,
         final String checkPath = getPackageFromFilePath(filePath) + "." + checkName + "Check";
         inputConfigBuilder.setCheckName(checkPath);
     }
+
+    private static void setCheckProperties(InputConfiguration.Builder inputConfigBuilder,
+                                      String filePath, List<String> lines)
+                    throws CheckstyleException {
+        int lineNo = 2;
+        for (String line = lines.get(lineNo); !line.contains("*/"); line = lines.get(++lineNo)) {
+            if (line.isEmpty()) {
+                continue;
+            }
+            try {
+                final int equalPosition = line.indexOf('=');
+                final String property = line.substring(0, equalPosition - 1);
+                if (line.contains("(default)")) {
+                    final int defaultEndPosition = line.indexOf(')');
+                    final String value = line.substring(defaultEndPosition + 1);
```

Comment:
you can simplify by
`final String value = line.substring(line.indexOf(')') + 1);`

Reply:
done